src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -8,79 +9,74 @@ import ChannelListScreen from '../ChannelListScreen';
 import ChannelListTabItem from '../../components/HorizontalTab/ChannelListTabItem';
 import HorizontalTab from '../../components/HorizontalTab';
 import Search from '../ChannelListScreen/elements/Search';
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
+import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useRootChannelListHook from '../../hooks/screen/useRootChannelListHook';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useLocalDatabaseHook` and `useUserAuthHook` is a good change as it seems to be replacing the deprecated `useProfileHook`. However, I don't see where `refreshAnonymousChannelList` and `refreshSignedChannelList` are being called in the new code. These were previously used when a tab was selected. If these refresh functions are still necessary, you might need to reintroduce them.



<!-- This is an auto-generated comment by OSS CodeRabbit -->